lib/widgets/molecules/profile_quick_view_card.dart

```diff
@@ -238,96 +206,93 @@ class ProfileQuickViewCard extends StatelessWidget {
-            image: avatarUrl != null
-                ? NetworkImage(avatarUrl!) as ImageProvider<Object>
-                : const AssetImage(Assets.blankAvatar),
-            width: 80.0,
-            height: 80.0, // Height of the avatar
-            fit: BoxFit.cover,
+          child: Divider(
+            color: theme.colorScheme.outlineVariant,
           ),
         ),
-        const SizedBox(width: Insets.paddingMedium),
-        Expanded(
-          child: Column(
-            mainAxisAlignment: MainAxisAlignment.start,
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: [
-              Text.rich(
-                TextSpan(
-                  text: fullName,
-                  style: theme.textTheme.labelSmall?.copyWith(
-                    height: 1.4,
-                    color: theme.colorScheme.onPrimaryContainer,
-                  ),
-                  children: [
-                    TextSpan(
-                      text: ' ${_userTypeTag(l10n)}',
-                      style: theme.textTheme.labelSmall?.copyWith(
-                        color: theme.colorScheme.onSurfaceVariant,
-                      ),
-                    )
-                  ],
-                ),
-              ),
-              const SizedBox(height: Insets.paddingExtraSmall),
-              Text(
-                location,
-                style: theme.textTheme.labelSmall?.copyWith(
-                  color: theme.colorScheme.secondary,
-                  fontWeight: FontWeight.w300,
-                ),
+        Row(
+          mainAxisAlignment: MainAxisAlignment.start,
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            ClipRRect(
+              borderRadius: BorderRadius.circular(
+                Radii.roundedRectRadiusMedium,
               ),
-              const Divider(
-                endIndent: Insets.paddingExtraLarge,
+              child: Image(
+                image: avatarUrl != null
+                    ? NetworkImage(avatarUrl!) as ImageProvider<Object>
+                    : const AssetImage(Assets.blankAvatar),
+                width: 80.0,
+                height: 80.0, // Height of the avatar
+                fit: BoxFit.cover,
               ),
-              if (companyText != null)
-                Text(
-                  companyText,
-                  style: theme.textTheme.labelSmall?.copyWith(
-                    color: theme.colorScheme.secondary,
+            ),
+            const SizedBox(width: Insets.paddingMedium),
+            Expanded(
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.start,
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  if (verizonDigitalReady == true)
+                    ProfileChip(
+                      text: l10n.verizonDigitalReady,
+                      verizonChip: true,
+                    ),
```

Comment:
The chip in general displays group memberships other than mentor/mentee. Verizon Digital Ready is just used as an example in the mock. The backend for this still needs some work, but we should at least have this chip be customizable to display whatever String we get from the query

Reply:
okay